README.md

```diff
@@ -390,6 +390,24 @@ Artistic License
 * `artistic2`
 
 Artistic License 2.0
+* `ccby`
```

Comment:
The abbreviations are the same that are used here:
https://creativecommons.org/licenses/?lang=en